### ISSUE TITLE
’‘’/tools/lane_evaluation‘’‘ not exists.

### ISSUE THREAD

**Issue by FANG-MING** [Turoad/CLRNet#23]
'''cd $CLRNET_ROOT/tools/lane_evaluation''', but the dir is not exists.

**Comment by MySuperSoul** [comment]
> `cd $CLRNET_ROOT/tools/lane_evaluation`, but the dir is not exists.

Yes, this is a mistake, you can ignore this step, we have reimplemented the eval metrics using python opencv.
